gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1499,6 +1499,28 @@ gst_tensors_config_from_structure (GstTensorsConfig * config,
   return TRUE;
 }
 
+/**
+ * @brief Parse caps and set tensors config (for other/tensors)
+ * @param config tensors config structure to be filled
+ * @param caps incoming capability
+ * @return TRUE/FALSE (if successfully configured, return TRUE)
+ */
+gboolean
+gst_tensors_config_from_cap (GstTensorsConfig * config, GstCaps * caps)
+{
+  GstStructure *structure;
+
+  if (!gst_caps_is_fixed (caps)) {
+    nns_logw ("GstCaps is not fixed");
+    return FALSE;
+  }
+
+  structure = gst_caps_get_structure (caps, 0);
+  gst_tensors_config_from_structure (config, structure);
+
+  return gst_tensors_config_validate (config);
```

Comment:
Suggestion:
`return gst_tensors_config_from_structure (config, structure) && gst_tensors_config_validate (config);`

Reply:
I didn't use the return value of `gst_tensors_config_from_structure`
Thank you 👍